openapi/pom.xml

```diff
@@ -102,6 +102,16 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
If you used the WebClient, there would be no need for this setting - maybe a follow up for this PR?

Reply:
The tests in OpenAPI predate the WebClient and I did not want to rewrite the tests...not at this moment, anyway. I have used the WebClient in the new tests in the `webserver/cors` module itself and even found a bug in it (which David has found a fix for)!